packages/realm/package.json

```diff
@@ -114,7 +115,11 @@
       "command": "cmake-js configure --debug --directory bindgen"
     },
     "bindgen:configure:browser": {
-      "command": "emcmake cmake -DCMAKE_BUILD_TYPE=Debug -S bindgen -B bindgen/build.browser -G Ninja"
+      "command": "emcmake cmake -DCMAKE_BUILD_TYPE=Debug -S bindgen -B bindgen/build.browser -G Ninja",
+      "files": [
```

Comment:
Did this fixed the caching issues with `wireit`?

Reply:
Yeah, thankfully I haven't had to delete it manually while iterating on this.